.github/actions/deploy-klt-on-cluster/action.yml

```diff
@@ -26,6 +26,10 @@ inputs:
     required: false
     description: "Use scheduling gates instead of scheduler"
     default: "false"
+  allowed-namespaces:
```

Comment:
I think we should have a better pipeline for helm configurations :/
otherwise everytime we will increase the matrix exponentially

Reply:
A possible hack (I am not really fond of this) would be to patch the conf before and after the set of tests under `simple-deployment-not-allowed`.

Reply:
ok, what do you suggest ? A separate pipeline will introduce extremely much duplications, also the matrix will be decreased in the future by removing the support for manifests installation option and also removing the support of scheduler -> this way we have only one parameter left to configure -> allowed-namespaces

Reply:
Also using the patch itself won't test the installation of the webhookConfiguration with the helm-charts itself, it will just test that the namespaces are not enabled for the webhook to kick in

Reply:
> Also using the patch itself won't test the installation of the webhookConfiguration with the helm-charts itself, it will just test that the namespaces are not enabled for the webhook to kick in

Depending on what you want to test. If you want to test the changes propagated by Helm into the manifests, then a separate pipeline is better. If you want to test the operator behavior the patch suffices.